docker/rust/release/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM stablex-binary
+FROM stablex-binary-private
```

Comment:
Should we maybe use public as the default here to avoid accidentally exposure of the private one?

Reply:
This PR intended to not change this logic, but just to keep it as before. The logic will be touched in https://github.com/gnosis/dex-services/pull/681